fcrepo-auth-common/src/main/java/org/fcrepo/auth/common/FedoraUserSecurityContext.java

```diff
@@ -37,6 +37,9 @@
     private static final Logger LOGGER = LoggerFactory
             .getLogger(FedoraUserSecurityContext.class);
 
+    // Defined here because FedoraJcrTypes is in fcrepo-http-api, which would create a cyclic dependency
+    protected static final String JCR_CONTENT = "jcr:content";
```

Comment:
Yes, if there is any problem with any module at all using the API, that is a massive architecture failure. the API should be essentially independent of _everything else_. I'm not saying it is, I'm saying that such is the condition for which we must be working.